packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -483,6 +483,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
         },
         {
           test: /\.[cm]?js$|\.tsx?$/,
+          resolve: {fullySpecified: false},
```

Comment:
```suggestion
          // The below is needed due to a bug in `@babel/runtime`. See: https://github.com/babel/babel/issues/12824
          resolve: {fullySpecified: false},
```

Let's add a comment explaining why this is needed.